.github/ISSUE_TEMPLATE.md

```diff
@@ -1,9 +1,30 @@
+This is the default new issue template! You may use the next section to enter a new user story, or use the last section to enter a new bug. Please delete this header and the other unused section before submitting this issue.
+
+---
+
 As X, I want Y so that Z.
 
-## Acceptance Criteria 
+## Acceptance Criteria
 
 - [ ] Verify
 
 ## Assumptions and Questions
 
-- 
+-
+
+---
+
+X is a bug because Y happens when I expect Z.
+
+## Steps to Reproduce
+
+1.
+1.
+
+## Expected Result
+
+-
+
+## Actual Result
+
+-
```

Comment:
I think one more section "Environment information". Could just be heading or could be a table with some default values:

service | provider | version
---------- | ----------- | --------------
cloudigrade | dev server | my-commit-hash
postgres | local openstack | default
rabbitmq | local openstack | default